### DISCUSSION TITLE
Xplorer v0.2.0

### DISCUSSION THREAD

**Discussion by kimlimjustin** [kimlimjustin/xplorer#175] [General]
**Introducing the second experimental version of Xplorer 🎉🥳 . See the assets to download this version and install.**

## Major Changes
* **Migrate from Electron to Tauri** (#173) (**breaking changes**) 🚀
* Introduce new icon <img src="https://user-images.githubusercontent.com/61906684/139273868-139bac4f-fbaa-4df0-bbb9-af752ceddfbf.png" width="20px">
* Improve Xplorer layout 
   <img src="https://github.com/kimlimjustin/xplorer/raw/master/docs/static/img/Xplorer_dark.png" width="300px">
* Improve performance and memory usage (experimental) 👨‍🔬
* Decrease installer size up to 90% (#173) 🚀
* Add go to parent directory ability  🚀
* Non-ASCII drive name supports 🚀
* Add ability to continue the previous session on startup 🚀
* More personalized settings ⚙️
* Supports previewing almost all programming languages with syntax highlighting 👁️‍🗨️
* Improve docs and locales 📃
* Improve some functions algorithms 🚀
* Update some shortcuts (**breaking changes**) 💥

## Detail Changes
* Xplorer CLI by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/43
* docs: fix typo in readme and small rephrase by @bitomic in https://github.com/kimlimjustin/xplorer/pull/44
* Fixed some typos in the documentation by @michaelgrigoryan25 in https://github.com/kimlimjustin/xplorer/pull/47
* German translation by @silas229 in https://github.com/kimlimjustin/xplorer/pull/51
* docs: fix typos and grammar on pages by @rushabh-wadkar in https://github.com/kimlimjustin/xplorer/pull/62
* fix: make close settings button clickable by @rushabh-wadkar in https://github.com/kimlimjustin/xplorer/pull/63
* bug: disable spell check on filepath input by @rushabh-wadkar in https://github.com/kimlimjustin/xplorer/pull/71
* bug: ignore keyboard shortcuts if filepath navigator is in focus by @rushabh-wadkar in https://github.com/kimlimjustin/xplorer/pull/73
* bug: make properties box exit entirely when closed by @rushabh-wadkar in https://github.com/kimlimjustin/xplorer/pull/74
* UI Enhancement by @dkhd in https://github.com/kimlimjustin/xplorer/pull/80
* update: enhance version page by @rushabh-wadkar in https://github.com/kimlimjustin/xplorer/pull/82
* Smaller workflows by @theblobscp in https://github.com/kimlimjustin/xplorer/pull/85
* fix: Favorites & Drives changed when changing the main layout and remove tilt effect by @dkhd in https://github.com/kimlimjustin/xplorer/pull/87
* Update install.md by @lolcabanon in https://github.com/kimlimjustin/xplorer/pull/88
* fix: File/Folder max character length by @dkhd in https://github.com/kimlimjustin/xplorer/pull/89
* Added french translation by @thomasbnt in https://github.com/kimlimjustin/xplorer/pull/91
* Modified Installation docs by @DTan13 in https://github.com/kimlimjustin/xplorer/pull/90
* Do not navigate if the path navigator is empty by @Turbinya in https://github.com/kimlimjustin/xplorer/pull/95
* Ignore arrow keys for select files if path navigator has focus by @Turbinya in https://github.com/kimlimjustin/xplorer/pull/98
* Upgrade sass by @theblobscp in https://github.com/kimlimjustin/xplorer/pull/96
* feat: Gitpod Support by @im-coder-lg in https://github.com/kimlimjustin/xplorer/pull/84
* Fix image icon in README.md by @AnilSeervi in https://github.com/kimlimjustin/xplorer/pull/100
* Change Preview-file docs link in Homepage by @AnilSeervi in https://github.com/kimlimjustin/xplorer/pull/101
* optimize doc index page slide show animation and refactor related React code by @XDean in https://github.com/kimlimjustin/xplorer/pull/104
* feat(shortcuts): navigation with additional mouse keys (#58) by @Turbinya in https://github.com/kimlimjustin/xplorer/pull/107
* Fix broken Github links to File Icon and File type by @AnilSeervi in https://github.com/kimlimjustin/xplorer/pull/108
* fix(locales): remove unnecessary explanation from the "русский язык" by @Turbinya in https://github.com/kimlimjustin/xplorer/pull/110
* chore(copyfiles): unused require by @Turbinya in https://github.com/kimlimjustin/xplorer/pull/111
* Improve Issue Templates by @stanleyowen in https://github.com/kimlimjustin/xplorer/pull/116
* chore: move jest to dev dependencies by @Turbinya in https://github.com/kimlimjustin/xplorer/pull/119
* update: README.md by @dkhd in https://github.com/kimlimjustin/xplorer/pull/118
* Update README, fix typo and sentences by @AnilSeervi in https://github.com/kimlimjustin/xplorer/pull/117
* fix(docs): Grammatical Error correction by @im-coder-lg in https://github.com/kimlimjustin/xplorer/pull/105
* Fix typo in homepage by @AnilSeervi in https://github.com/kimlimjustin/xplorer/pull/121
* Use `<code>` instead of `<kbd>` by @AnilSeervi in https://github.com/kimlimjustin/xplorer/pull/124
* feat: theme customization by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/127
* Remove duplicate feature lists by @AnilSeervi in https://github.com/kimlimjustin/xplorer/pull/128
* Style properties by @im-tridevsharma in https://github.com/kimlimjustin/xplorer/pull/129
* fix: cannot be opened on Windows 10 (resolve #138) by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/140
* Added Active Tab Styling in Settings by @stanleyowen in https://github.com/kimlimjustin/xplorer/pull/141
* Custom theme examples by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/142
* fix: files with file name more than 30 characters won't able to be operated by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/130
* Properties Style by @im-tridevsharma in https://github.com/kimlimjustin/xplorer/pull/131
* fix: cannot open file/directory with long characters by @dkhd in https://github.com/kimlimjustin/xplorer/pull/132
* Properties style by @dkhd in https://github.com/kimlimjustin/xplorer/pull/133
* fix: window manager buttons too cramped by @dkhd in https://github.com/kimlimjustin/xplorer/pull/134
* Reduce duplicated code by @AlexHladin in https://github.com/kimlimjustin/xplorer/pull/135
* Fix Slideshows not displaying images for other languages by @AnilSeervi in https://github.com/kimlimjustin/xplorer/pull/143
* feat: add size property for folder (#76) by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/145
* feat: support non-ascii drive character (#114) by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/144
* feat: hover effect by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/149
* perf: reduce the number of left-click event listeners on files by @Turbinya in https://github.com/kimlimjustin/xplorer/pull/148
* refactor: Performance by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/154
* Add Ukrainian language by @AlexHladin in https://github.com/kimlimjustin/xplorer/pull/155
* Translated to Japanese. by @javaboy-github in https://github.com/kimlimjustin/xplorer/pull/150
* refactor: file type and file thumbnail code by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/156
* Update pt-BR.json by @dgsmiley18 in https://github.com/kimlimjustin/xplorer/pull/160
* Update Ukrainian translation by @AlexHladin in https://github.com/kimlimjustin/xplorer/pull/161
* Update Ukrainian translation by @AlexHladin in https://github.com/kimlimjustin/xplorer/pull/162
* Convert code blocks to fenced code blocks for better accessibility by @AnilSeervi in https://github.com/kimlimjustin/xplorer/pull/165
* Refactor NewFile and NewFolder functions. by @AlexHladin in https://github.com/kimlimjustin/xplorer/pull/167
* Refactoring select file operation by @AlexHladin in https://github.com/kimlimjustin/xplorer/pull/166
* fix: Add missed role for submenu items by @AlexHladin in https://github.com/kimlimjustin/xplorer/pull/168
* Precalculate path prefix for fs operations by @AlexHladin in https://github.com/kimlimjustin/xplorer/pull/170
* Rebranding of Xplorer by @kimlimjustin in https://github.com/kimlimjustin/xplorer/pull/173

## New Contributors 🎉
* @bitomic made their first contribution in https://github.com/kimlimjustin/xplorer/pull/44
* @michaelgrigoryan25 made their first contribution in https://github.com/kimlimjustin/xplorer/pull/47
* @silas229 made their first contribution in https://github.com/kimlimjustin/xplorer/pull/51
* @rushabh-wadkar made their first contribution in https://github.com/kimlimjustin/xplorer/pull/62
* @dkhd made their first contribution in https://github.com/kimlimjustin/xplorer/pull/80
* @theblobscp made their first contribution in https://github.com/kimlimjustin/xplorer/pull/85
* @lolcabanon made their first contribution in https://github.com/kimlimjustin/xplorer/pull/88
* @thomasbnt made their first contribution in https://github.com/kimlimjustin/xplorer/pull/91
* @DTan13 made their first contribution in https://github.com/kimlimjustin/xplorer/pull/90
* @Turbinya made their first contribution in https://github.com/kimlimjustin/xplorer/pull/95
* @AnilSeervi made their first contribution in https://github.com/kimlimjustin/xplorer/pull/100
* @XDean made their first contribution in https://github.com/kimlimjustin/xplorer/pull/104
* @im-tridevsharma made their first contribution in https://github.com/kimlimjustin/xplorer/pull/129
* @AlexHladin made their first contribution in https://github.com/kimlimjustin/xplorer/pull/135
* @javaboy-github made their first contribution in https://github.com/kimlimjustin/xplorer/pull/150
* @dgsmiley18 made their first contribution in https://github.com/kimlimjustin/xplorer/pull/160

**Please note that this is a development version, not stable yet. Anything here might be changed in the future.**

Join our discord [here](https://discord.gg/kK7rwxPt). 
_Please drop a star ⭐ if you like this project_

**Full Changelog**: https://github.com/kimlimjustin/xplorer/compare/v0.1.0...v0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kimlimjustin/xplorer/releases/tag/v0.2.0'>Xplorer v0.2.0</a>.</em>

**Comment by thomasbnt** [comment]
🎉🎉🎉🥳

**Comment by lg-nineoneninethreenine** [comment]
Excellent! The icon being an emoji is super cool!

**Reply by kimlimjustin** [reply]
Thanks!  Actually, it's just a markdown trick haha, use <img> tag with width attribute

**Reply by lg-nineoneninethreenine** [reply]
Ingenious!

**Reply by lg-nineoneninethreenine** [reply]
I think this can be added to Primer Emojis, it's cool!

**Reply by lg-nineoneninethreenine** [reply]
```html
<img alt="" width="20px" src="https://user-images.githubusercontent.com/61906684/139273868-139bac4f-fbaa-4df0-bbb9-af752ceddfbf.png">
```
<img alt="" width="20px" src="https://user-images.githubusercontent.com/61906684/139273868-139bac4f-fbaa-4df0-bbb9-af752ceddfbf.png">
Code is HTML, and Markdown gets converted to HTML. An example is Jekyll!

**Reply by joshlimitless** [reply]
Hi! I seem to have an issue running the program on a Mac, and joining the Discord channel - the verification bot seems to be broken.

**Reply by kimlimjustin** [reply]
> Hi! I seem to have an issue running the program on a Mac, and joining the Discord channel - the verification bot seems to be broken.

Hello, I saw you on Discord and waved to you, are you able to chat in?

**Comment by AnilSeervi** [comment]
awesome!! 🎉
